Fix Generate and Copy Button

AmreshSinha opened the pull request:
# Fixed Generate and Copy Button

https://github.com/surajshende247/Password-Knitter/issues/5

## Description

I have fixed the issue of Generate and Copy button which was not working earlier.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)


## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have checked my code and corrected any misspellings

## Additional Context

I want to know how the green button functions (like on selecting it removes that particular format). There is no documentation on that. You can open a new issue with the functions given of that green button.

surajshende247 reviewed (approved):
Good Work 👍

I suggest adding comments in JavaScript code

AmreshSinha commented:
Oh my bad. These were the changes which I did:
- Added readonly attribute
- Fixed Copy and Generate Button
- Change Text State of Buttons on clicking
